models/tracker/src/index.ts

```diff
@@ -1020,19 +1020,19 @@ export function createModel (builder: Builder): void {
             label: tracker.string.MyIssues,
             icon: tracker.icon.MyIssues,
             component: tracker.component.MyIssues
-          } //,
-          // {
-          //   id: 'all-issues',
-          //   position: 'top',
-          //   label: tracker.string.AllIssues,
-          //   icon: tracker.icon.Issues,
-          //   component: tracker.component.IssuesView,
-          //   componentProps: {
-          //     query: { '$lookup.space.archived': false },
-          //     space: undefined,
-          //     title: tracker.string.AllIssues
-          //   }
-          // }
+          },
+          {
+            id: 'all-issues',
+            position: 'top',
+            label: tracker.string.AllIssues,
+            icon: tracker.icon.Issues,
+            component: tracker.component.IssuesView,
+            componentProps: {
+              query: { '$lookup.space.archived': false },
```

Comment:
For discuss, maybe we can use spaceSecurityMiddleware to hide archive spaces in future?

Reply:
They still should be available for Vacancies, so better to have view option to hide/show them.